### DISCUSSION TITLE
Outreachy applicant

### DISCUSSION THREAD

**Discussion by esraa-abdelmaksoud** [ModECI/MDF#238] [General]
Hello,
My name is Esraa Abdelmaksoud. I am currently pursuing a professional master's in Software Engineering in Egypt. My thesis is about data extraction from unstructured medical documents to build up electronic health records in countries that don't support them yet. Though I was working in marketing in the past for renowned companies, I decided to follow my passion to add value to the healthcare industry.

I already got my first pull request merged here for a minor problem, but I will be happy to continue contributing to this valuable project.

**Comment by pgleeson** [comment]
Hi @esraa-abdelmaksoud. Thanks for the PR. Merged.

**Reply by esraa-abdelmaksoud** [reply]
Thanks for your effort! :)
